Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -2159,16 +2159,35 @@ void WebPageProxy::viewWillStartLiveResize()
         return;
 
     closeOverlayedViews();
+    
+#if ENABLE(ASYNC_SCROLLING) && PLATFORM(COCOA)
+    if (m_scrollingCoordinatorProxy)
```

Comment:
I think it would be more natural to plumb these through the `DrawingAreaProxy` and have the `RemoteLayerTreeDrawingAreaProxy` overrides call into the scrolling coordinator.

Reply:
Fixed.